changelog/fragments/ginkgo-v2-changes.yaml

```diff
@@ -0,0 +1,16 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      updated ginkgo to ginkgo/v2
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "change"
+
+    # Is this a breaking change?
+    breaking: false
```

Comment:
Since there are a lot more changes going in here that affect users we need to add more changelog entries that include the changes made and any associated migration guides.

Reply:
Yes, will add separate changelogs with migration guides.